src/main/java/com/atlassian/bitbucket/jenkins/internal/trigger/RetryingWebhookHandler.java

```diff
@@ -120,7 +122,8 @@ private BitbucketWebhook registerWithRetry(
                 .orElse(null);
 
         if (result == null) {
-            BitbucketCredentials credentials = jenkinsToBitbucketCredentials.toBitbucketCredentials(jobCredentials);
+            // TODO Fixing this to use context breaks A LOT of tests. Deal with this later.
```

Comment:
I'll address this in a separate PR. Expecting to remove this code path altogether- we currently assume the http token has admin access to any linked repository. I would argue if it doesn't, that's a config error.

Reply:
Maybe document the outcome of our discussion here? (I cannot quite remember what we decided in this regard)